src/kotlin_script.cpp

```diff
@@ -128,15 +128,15 @@ KtClass* KotlinScript::get_kotlin_class() const {
 Variant KotlinScript::_new(const Variant **p_args, int p_argcount, Variant::CallError &r_error) {
     r_error.error = Variant::CallError::CALL_OK;
 
-    Object *owner = ClassDB::instance(get_kotlin_class()->super_class);
+    Object* owner{ClassDB::instance(get_kotlin_class()->base_godot_class)};
```

Comment:
From what I understand, we just need to pass in the user defined inherited class?

Reply:
Also, does the order of registration matter? i.e can `ClassInheritanceChild` be registered before `ClassInheritanceParent`?

Reply:
Nop, thing is with registration trick, godot does not even have to know the user defined super class.  
All inherited methods are also registered in children class.  
This makes more functions registered by classes, but avoid cpu time to retrieve methods throught inheritance.  
With this, registration order does not matter at all. Godot just have to know the base godot class.  
Basically we take advantage of JVM vtable.